js/spellChecker/index.js

```diff
@@ -24,7 +25,8 @@ class SpellCheckHelper {
         // In windows we need to implement RxJS observable
         // in order to switch language dynamically
         if (!isMac) {
-            this.spellCheckHandler.switchLanguage('en-US');
+            const sysLocale = remote.app.getLocale() || 'en-US';
```

Comment:
how does lang get changed dynamically?

Reply:
`spell-checker` module uses `Observable.FromEvent` to attach EventListener for all the inputs in the document and listens for changes to detect language and to switch it dynamically based on the input text.

As `RTE` is not an input element the language will not dynamically change which indicates all the words as misspelt